web/beacon-app/src/features/topics/components/QueryForm.tsx

```diff
@@ -0,0 +1,37 @@
+import { Trans } from '@lingui/macro';
+import { Button, TextField } from '@rotational/beacon-core';
+
+type QueryFormProps = {
+  defaultEnSQL: string;
+  queryHandler: (query: string) => void;
+};
+
+const QueryForm = ({ defaultEnSQL, queryHandler }: QueryFormProps) => {
+  const onQuery = (e: any) => {
+    e.preventDefault();
+    queryHandler(e.target.value);
+  };
+
+  return (
+    <div className="mt-4 flex space-x-2">
+      <TextField
+        labelClassName="font-semibold"
+        type="search"
+        defaultValue={defaultEnSQL}
+        onChange={onQuery}
+        name="query"
+        fullWidth
+      ></TextField>
+      <div className="flex space-x-2">
+        <Button variant="secondary" onClick={onQuery}>
```

Comment:
Should `type="submit"` be added here?

Reply:
I think this one is not needed since you are going to add Formik. I added this one just for test purposes.